component/sections/cookies/variant_a.js

```diff
@@ -1,72 +1,46 @@
-import React, { useState } from "react";
+import React from "react";
+import { setCookie, getCookie } from "../../../utils/cookies"
+
 
 function VariantA({ title, text, button1, button2 }) {
-  //to hide cookie component
-  const [showCookie, setShowCookie] = useState(true);
+  let cookieExists = getCookie();
+  const [showCookie, setShowCookie] = React.useState(cookieExists);
```

Comment:
Use lazy initialization. For example, see below: 
![image](https://user-images.githubusercontent.com/6467262/126478833-9408a616-b9f7-4309-be4b-83044bead948.png)

Read more about it here: https://kentcdodds.com/blog/use-state-lazy-initialization-and-function-updates

or https://reactjs.org/docs/hooks-reference.html#lazy-initial-state

Reply:
This is definitely something I missed. Such a good read from given sources. Thank you very much!